### ISSUE TITLE
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.app.Activity.getPackageName()' on a null object reference

### ISSUE THREAD

**Issue by thegdznet** [vinzscam/react-native-file-viewer#117]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-file-viewer@2.1.4` for the project I'm working on based on a crash log collected via AppCenter.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.app.Activity.getPackageName()' on a null object reference

com.vinzscam.reactnativefileviewer.RNFileViewerModule.open RNFileViewerModule.java:55
java.lang.reflect.Method.invoke Method.java
com.facebook.react.bridge.JavaMethodWrapper.invoke JavaMethodWrapper.java:372
com.facebook.react.bridge.JavaModuleWrapper.invoke JavaModuleWrapper.java:151
com.facebook.react.bridge.queue.NativeRunnable.run NativeRunnable.java
android.os.Handler.handleCallback Handler.java:883com.vinzscam.reactnativefileviewer.RNFileViewerModule.open RNFileViewerModule.java:55
java.lang.reflect.Method.invoke Method.java
com.facebook.react.bridge.JavaMethodWrapper.invoke JavaMethodWrapper.java:372
com.facebook.react.bridge.JavaModuleWrapper.invoke JavaModuleWrapper.java:151
com.facebook.react.bridge.queue.NativeRunnable.run NativeRunnable.java
android.os.Handler.handleCallback Handler.java:883
```

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38.bin b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38.bin
new file mode 100644
index 0000000..5ff383e
--- /dev/null
+++ b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38.bin
@@ -0,0 +1 @@
+o/debug
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/BuildConfig.dex b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/BuildConfig.dex
new file mode 100644
index 0000000..cbeabb4
Binary files /dev/null and b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/BuildConfig.dex differ
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/FileProvider.dex b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/FileProvider.dex
new file mode 100644
index 0000000..9d72548
Binary files /dev/null and b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/FileProvider.dex differ
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule$1.dex b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule$1.dex
new file mode 100644
index 0000000..0ff1a95
Binary files /dev/null and b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule$1.dex differ
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule.dex b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule.dex
new file mode 100644
index 0000000..039c738
Binary files /dev/null and b/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vinzscam/reactnativefileviewer/RNFileViewerModule.dex differ
diff --git a/node_modules/react-native-file-viewer/android/build/.transforms/26621e9c688a9714df136801936f5f38/debug/com/vin

**Comment by vinzscam** [comment]
Hi!
thank you for reporting the issue.
Do you have more information about the changes you've made for fixing the issue? Unfortunately the diff above doesn't show anything useful.

**Comment by vinzscam** [comment]
thanks! Fixed by #124

**Comment by thegdznet** [comment]
Thanks a lot!
